### PR TITLE
Update stable_diffusion notebook with steps for local inference.

### PR DESCRIPTION
**Pull request by dstnluong-google**
Add local inference with Workbench for stable_diffusion.
<br><br><br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb [hunk 1]
```diff
@@ -24,6 +24,7 @@
       ]
```
[the remark this answers is not in the document]
Comment (dstnluong-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:24]: <p>Done.</p>

[the remark this answers is not in the document]
Comment (dstnluong-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:24]: <p>Done.</p>

[the remark this answers is not in the document]
Comment (dstnluong-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:24]: <p>Done.</p>

[the remark this answers is not in the document]
Comment (dstnluong-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:24]: <p>using print gives "&lt;PIL.Image.Image image mode=RGB size=768x768 at 0x7F44F1997A50&gt;", changing to display</p><p> </p>

[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:24]: did you mean "inference dockers". You saw local prediction -- what does a training docker file have to do with this?

[the remark this answers is not in the document]
Comment (dstnluong-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb:24]: Changed to "serving dockers"

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**